Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1568,7 +1568,8 @@ def lhegensim2017(fragment,howMuch):
                                '--data':'',
                                '--era':'Run2_HI',
                                '--customise':'RecoLocalTracker/SiStripZeroSuppression/customiseHybrid.addHybridEmulationBeforeRepack',
-                               '--processName':'EMULATEHYBRID',    
+                               '--processName':'EMULATEHYBRID',   
+                               '--outputCommands':'"drop FEDRawDataCollection_*__HLT", "drop FEDRawDataCollection_*__LHC"', 
```

Comment:
the space between the two drop statements is still here
it should be ```'"drop FEDRawDataCollection_*__HLT","drop FEDRawDataCollection_*__LHC"'```

Reply:
and it causes the step4 of 140.55 to fail:

09:48 cmsdev15 958> edmDumpEventContent step3.root 
Type                      Module                    Label     Process           
--------------------------------------------------------------------------------
edm::TriggerResults       "TriggerResults"          ""        "HLT"             
trigger::TriggerEvent     "hltTriggerSummaryAOD"    ""        "HLT"             
FEDRawDataCollection      "rawDataCollector"        ""        "LHC"             
FEDRawDataCollection      "hybridRawDataRepacker"   ""        "EMULATEHYBRID"   
edm::TriggerResults       "TriggerResults"          ""        "EMULATEHYBRID"   
FEDRawDataCollection      "rawDataRepacker"         ""        "REHLT"           
edm::TriggerResults       "TriggerResults"          ""        "REHLT"          

Furthermore, should not this be FEDRawDataCollection_*_*_HLT ?